.github/workflows/edgar-feed.yaml

```diff
@@ -0,0 +1,73 @@
+name: Rust CI
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    working-directory: edgar-feed
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  linting:
+    name: linting with clippy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+      - uses: actions/checkout@v2
+      - name: Install Rust toolchain
+      - uses: actions-rs/toolchain@v1
+        with:
+          components: clippy
+          toolchain: stable
+          override: true
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          args: -- -D warnings
+
+  security-audit:
+    name: cargo dependency audit
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+      - uses: actions/checkout@v1
+      - name: Install Rust toolchain and run audit check
+      - uses: actions-rs/audit-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+  formatting:
+    name: code formatting with rustfmt
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+      - uses: actions/checkout@v2
+      - name: Install Rust toolchain
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+          components: rustfmt
+      - name: formatting with rustfmt
+      - uses: actions-rs/cargo@v1
+        with:
+          command: fmt
+          args: --all -- --check
+
+  test:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+      - uses: actions/checkout@v2
+      - name: Install Rust toolchain
+      - uses: actions-rs/toolchain@v1
```

Comment:
Yeah, unless I'm reading this wrong, the `name` and `uses` both need to be under the same list element.